redis/src/lib.rs

```diff
@@ -0,0 +1,100 @@
+//! Redis support for the `bb8` connection pool.
+#![deny(missing_docs, missing_debug_implementations)]
+
+pub extern crate bb8;
+pub extern crate redis;
+
+extern crate futures;
+extern crate tokio_core;
+
+use futures::{Future, IntoFuture};
+use tokio_core::reactor::Handle;
+
+use redis::async::Connection;
+use redis::{Client, RedisError};
+
+use std::io;
+use std::option::Option;
+
+type Result<T> = std::result::Result<T, RedisError>;
+
+/// `RedisPool` is a convenience wrapper around `bb8::Pool` that hides the fact that
+/// `RedisConnectionManager` uses an `Option<Connection>` to smooth over the API incompatibility.
+#[derive(Debug)]
+pub struct RedisPool {
+    // The `bb8::Builder` that manages the pool.
```

Comment:
Just "the underlying `bb8::Pool`" would be fine, though given that this is obvious and private idk that it's worth commenting.